orderbook/openapi.yml

```diff
@@ -240,6 +240,41 @@ paths:
                 $ref: "#/components/schemas/FeeInformation"
         404:
           description: Token non-existent or not connected to native token
+  /api/v1/price/{baseToken}/{quoteToken}:
+    get:
+      description: |
+        The estimated price for selling an optionally specified amount of baseToken or buying that amount of quote token. Price is denominated in quote token.
```

Comment:
Ok, so it's optional just to get the mid market price, or some market price calculation. If this is the case, I would consider to make it mandatory and be explicit on what is exactly this price. 

From the definition, I would think is a "spot price", since we give the amount and the direction of the order.